lib/java_buildpack/framework/jrebel_agent.rb

```diff
@@ -24,36 +24,51 @@ module Framework
     # Encapsulates the functionality for enabling zero-touch JRebel support.
     class JrebelAgent < JavaBuildpack::Component::VersionedDependencyComponent
 
+      def initialize(context, &version_validator)
+        super(context, &version_validator)
+        @component_name = 'JRebel Agent'
+      end
+
       # (see JavaBuildpack::Component::BaseComponent#compile)
       def compile
-        download_zip false
-        FileUtils.mv(download_location + 'jrebel.jar', @droplet.sandbox + jar_name)
-        FileUtils.remove_dir(download_location, true)
+        download_zip(true, @droplet.sandbox + 'install')
```

Comment:
Is there any reason that this can't simply be unzipped into the `@droplet.sandbox` directory?  If it can't be, then `@droplet.sandbox + 'install'` needs to be extracted to a function and re-used on line 40.